addons/purchase_stock/tests/test_reordering_rule.py

```diff
@@ -794,7 +794,7 @@ def test_multi_lingual_orderpoints(self):
         orderpoint.with_user(french_user).action_replenish()
         self.assertRecordValues(po_line, [{"name": "[A] produit en français", "product_qty": 9.0}])
         self.assertEqual(len(po_line.order_id.order_line), 1)
-        self.assertRecordValues(po_line.move_dest_ids, [{"product_uom_qty": 9.0}])
+        self.assertEqual(po_line.move_dest_ids.mapped("product_uom_qty"), [5.0, 4.0])
```

Comment:
```suggestion
        self.assertRecordValues(po_line.move_dest_ids, [{"product_uom_qty": 5.0}, {"product_uom_qty": 4.0}])
```

We can avoid mapping.